src/Avoidable.js

```diff
@@ -84,21 +83,24 @@ const Avoidable = ({
     let itemPosition = 0;
     let doesFitScreen = true;
     let shouldMove = !contextAware;
-    const safeAreaScreenHeight = Platform.OS === 'ios' ?
-      screenHeight - keyboardHeight :
-      safeAreaHeight;
-    const contentHeight = layoutMap[Object.keys(layoutMap).length - 1]?.y -
+    let contentHeight = layoutMap[Object.keys(layoutMap).length - 1]?.y -
       layoutMap[focusedField]?.y +
       layoutMap[Object.keys(layoutMap).length - 1]?.height +
```

Comment:
When `height` is undefined `contentHeight` will be `NaN`. You should handle this case. Same thing for `y`. (in case you didn't handle it elsewhere)